bin/helpers/capabilityHelper.js

```diff
@@ -42,18 +42,42 @@ const caps = (bsConfig, zip) => {
       reject("Test suite is empty");
     }
 
+    // Inferred settings
+    if (bsConfig.connection_settings && bsConfig.connection_settings.local_mode_inferred) {
+      obj.local_mode_inferred = bsConfig.connection_settings.local_mode_inferred;
+    }
+
+    if (bsConfig.connection_settings && bsConfig.connection_settings.local_inferred) {
+      obj.local_inferred = bsConfig.connection_settings.local_inferred;
+    }
+
+    if (bsConfig.connection_settings && bsConfig.connection_settings.sync_inferred) {
+      obj.sync_inferred = bsConfig.connection_settings.sync_inferred;
+      logger.info('Setting "sync" mode to enable Local testing.');
+    }
+
     // Local
     obj.local = false;
-    if (bsConfig.connection_settings && bsConfig.connection_settings.local === true) obj.local = true;
-    logger.info(`Local is set to: ${obj.local} (${obj.local ? Constants.userMessages.LOCAL_TRUE : Constants.userMessages.LOCAL_FALSE})`);
+    if (bsConfig.connection_settings && bsConfig.connection_settings.local === true) {
+      obj.local = true;
+    }
+
+    obj.localMode = null;
+    // Local Mode
+    if (obj.local === true && bsConfig.connection_settings.local_mode) {
+      obj.localMode = bsConfig.connection_settings.local_mode;
```

Comment:
I Node we generally follow camel-case instead of snake-case. Hence, have written it as localMode. All other variables in the file too follow camelCase.

Reply:
I think it's the data that we submit to the API and not a var/instance in code. Either way is fine.